src/client/Microsoft.Identity.Client/AppConfig/CiamAuthorityHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    internal class CiamAuthorityHelper
+    {
+        public static readonly string CiamMetadata = "{\"api-version\": \"1.1\",\"metadata\": [{\"preferred_network\": \"login.windows.net\",\"preferred_cache\": \"login.windows.net\",\"aliases\": [\"login.windows.net\",\"login.ciamlogin.com\"]}]}";
```

Comment:
I'm surprised that we still have `login.windows.net`, and not `login.microsoftonline.com` (it's targetting the v2.0 endpoint). Did you look in OIDC metadata what the issuer will be?

Reply:
Hmm, the [OIDC metadata](https://msidlabciam1.ciamlogin.com/msidlabciam1.onmicrosoft.com/v2.0/.well-known/openid-configuration?dc=ESTS-PUB-EUS-AZ1-FD000-TEST1&ciamhost=true) shows the issuer as `https://login.microsoftonline.com/tenant_guid/v2.0` (while the endpoints are `https://tenant.ciamlogin.com/tenant_guid/...`). Such a discrepancy does not affect MSAL, but is it a problem that we shall let ESTS folks know, @jmprieur ?

Regardless, I agree that we shall not need `login.windows.net` here.

Reply:
Removed `login.windows.net`